dotnetv3/IAM/README.md

```diff
@@ -1,99 +1,68 @@
-# AWS Identity and Access Management (IAM) code examples for .NET
+# IAM code examples for .NET
```

Comment:
for .NET --> for the SDK for .NET

(This also appears in other files.)

Reply:
Fixed in all cases.